telesign/phoneid.py

```diff
@@ -23,3 +31,78 @@ def phoneid(self, phone_number, **params):
         """
         return self.post(PHONEID_RESOURCE.format(phone_number=phone_number),
                          **params)
+
+    def _execute(self, method_function, method_name, resource, **params):
```

Comment:
Would prefer to see this incorporated into the base class RestClient function so that other endpoints can take advantage of it.